### ISSUE TITLE
Prisma example no longer works

### ISSUE THREAD

**Issue by kawadumax** [denoland/examples#15]
## Environment 
- WSL (Ubuntu)
- Deno 1.32.5

The following code results in an error when executed:

```shell
deno run --allow-read --allow-env --allow-write npm:prisma@^4.5 init
✅ Granted sys access to "osRelease".
✅ Granted run access to "/bin/sh".
✅ Granted sys access to "uid".
> Downloading Prisma engines for Node-API for debian-openssl-3.0.x [                    ] 0%┌ ⚠️  Deno requests net access to "bina✅ Granted net access to "binaries.prisma.sh".
> Downloading Prisma engines for Node-API for debian-openssl-3.0.x [=================== ] 93%error: Uncaught (in promise) TypeError: context.digest is not a function
    at Hash.flush [as _flush] (ext:deno_node/internal/crypto/hash.ts:47:35)
    at Hash.Ot [as _final] (ext:deno_node/_stream.mjs:11:59823)
    at Fa (ext:deno_node/_stream.mjs:11:51130)
    at Ua (ext:deno_node/_stream.mjs:11:51274)
    at mt (ext:deno_node/_stream.mjs:11:51350)
    at Hash.S.end (ext:deno_node/_stream.mjs:11:50523)
    at he.u (ext:deno_node/_stream.mjs:11:36397)
    at Object.onceWrapper (ext:deno_node/_events.mjs:502:28)
    at he.emit (ext:deno_node/_events.mjs:391:35)
    at Ea (ext:deno_node/_stream.mjs:11:43461)
```

**Comment by kawadumax** [comment]
related issue: https://github.com/denoland/deno/issues/18785

**Comment by kawadumax** [comment]
For the time being, `prisma init` was able to work by downgrading deno, in my environment.
```shell
deno upgrade --version 1.32.4
deno run --allow-read --allow-env --allow-write npm:prisma init
```

**Comment by petamoriken** [comment]
@kawadumax Maybe this issue was fixed in Deno v1.33

**Comment by kawadumax** [comment]
as node 1.33 released, so this issue is fixed.
